Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.h

```diff
@@ -53,7 +53,7 @@ enum class PickerDismissalReason : uint8_t;
 - (NSArray<NSString *> *)currentAvailableActionTitles;
 - (NSArray<NSString *> *)acceptedTypeIdentifiers;
 
-+ (std::pair<RetainPtr<NSURL>, RetainPtr<NSURL>>)_copyToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager;
++ (std::tuple<BOOL, RetainPtr<NSURL>, RetainPtr<NSURL>>)_moveToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager asCopy:(BOOL)asCopy;
```

Comment:
Nit: I have a general disinclination for `std::tuple` interfaces because we drop information that could be useful when doing a quick pass through the code...

What do you think of representing this as a structure instead?